boards/arm/nrf52840_pca10090/nrf52840_pca10090_defconfig

```diff
@@ -7,6 +7,9 @@ CONFIG_BOARD_NRF52840_PCA10090=y
 # Enable MPU
 CONFIG_ARM_MPU=y
 
+# enable GPIO
```

Comment:
Since this board requires GPIO to function, add 'select GPIO' in boards/arm/nrf52840_pca10090/Kconfig.board and remove setting it in the defconfig.

Reply:
Does the board actually require GPIO to function, or does the board init code just not properly handle the case that GPIO is not enabled?

Reply:
GPIO is required by this board. When the driver is not enabled, this board's initialization code cannot configure the on-board switches (that provide routing of certain signals) according to the configuration set in Kconfig, hence the code become useless.